include/wx/msw/mimetype.h

```diff
@@ -62,6 +62,9 @@ class WXDLLIMPEXP_BASE wxFileTypeImpl
     // this is called  by Associate
     bool SetDescription (const wxString& desc);
 
+    // this is called  by Associate
+    void EnableNotify(bool value);
```

Comment:
I'd rather flip the logic and call this method `SuppressNotifications()` (and also probably with `MSW` prefix to emphasize that it's MSW-specific) because the default state of the notifications is to be enabled.